src/routes/comment.py

```
@@ -70,7 +70,7 @@ async def get_comments_by_article(article_id: int, session: SessionDep, request:
 @comment_general_router.get('/{id}', response_model=CommentResponse)
 async def get_comment_by_id(id: int, session: SessionDep, request: Request) -> CommentResponse:
     comment = session.get(Comment, id)
-    if not comment: raise HTTPException(status_code=404, detail=f"Comment {id} does not exist")
+    if not comment:raise HTTPException(status_code=404, detail=f"Comment {id} does not exist")
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)